java-file-formats/word-library/working-with-bookmarks.md

```diff
@@ -406,12 +407,12 @@ As per Microsoft Word behavior, you cannot replace the bookmark contents when th
 
 Case 1
 
-![Bookmark start and end present in different tables](WorkingwithBookmarks_images/WorkingwithBookmarks_img1.jpeg)
+![Bookmark start and end present in different tables](workingwithbookmarks_images/java-file-formats-word-library-different-tables.jpeg)
```

Comment:
why you mention file-format in image name?
java is the platform, word library is control name.
Don't use file format. revert the changes in all places.

Reply:
Reverted